tools/data/mit/preprocess_data.sh

```diff
@@ -9,7 +9,7 @@ fi
 
 cd ${DATA_DIR}
 
-wget http://data.csail.mit.edu/soundnet/actions3/split1/Moments_in_Time_Raw.zip
+# Download the Moments_in_Time_Raw.zip here manually
```

Comment:
Maybe leave a runnable api for users who assume runnable:

$DOWNLOAD_URL=$1
wget  $DOWNLOAD_URL Moments_in_Time_Raw.zip

And the zip name may change so manually assign it `Moments_in_Time_Raw.zip`

Reply:
I'm not sure if the url can be downloaded using wget. For example, the google drive url can not be downloaded directly.

Reply:
Not sure, either. Let's block this til I got the URL from the official site?

Reply:
I found it's OK to use wget.